src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,44 @@
 package mate.academy;
 
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.RejectedExecutionException;
+import java.util.concurrent.TimeUnit;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private static final int USERS_COUNT = 10;
+    private static final int TIMEOUT_SECONDS = 10;
+    private final CopyOnWriteArrayList<EventListener> listeners = new CopyOnWriteArrayList<>();
```

Comment:
Better to use abstraction
```suggestion
    private final List<EventListener> listeners = new CopyOnWriteArrayList<>();
```